scripts/lint/lint.sh

```diff
@@ -57,7 +57,7 @@ function run_tests() {
 
   if [[ -z "$FAST" ]]; then
     run "mypy" mypy $PYTHON_FILES
-    run "codespell" codespell $ALL_FILES
+    run "codespell" codespell $ALL_FILES -L Wen
```

Comment:
Do it like https://github.com/sunnypilot/opendbc/commit/edba9e8fb6576a0f9e5a5a4f225fc78f45758359

```suggestion
    run "codespell" codespell $ALL_FILES
```